supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/api/PipelinesApiRequestManager.scala

```diff
@@ -328,7 +347,15 @@ object PipelinesApiRequestManager {
     override def getMessage: String = e.getMessage
   }
 
-  class PAPIApiException(val e: Throwable) extends RuntimeException(e) with CromwellFatalExceptionMarker {
-    override def getMessage: String = "Unable to complete PAPI request"
+  sealed trait PAPIApiException extends RuntimeException with CromwellFatalExceptionMarker {
+    def e: Throwable
```

Comment:
ToL: (Really, just semi-ranting)
- I don't like single letter variables, let alone `def`s
- `Throwable` means shutdown the JVM. `Exception` is a handled error. I don't know follow why Scala loves throwing these things around so carelessly. Maybe just because it's in the name.

Reply:
I've always viewed the love of `Throwable` as a java-in-general thing, part of the cargo culting of "always use the highest abstraction possible in interfaces"